public/js/loaders/sprite.js

```diff
@@ -17,22 +17,22 @@ export function loadSpriteSheet(name) {
         if (sheetSpec.tiles) {
             sheetSpec.tiles.forEach(tileSpec => {
                 sprites.defineTile(
-                    tileSpec.name,
+                    tileSpec.style,
                     tileSpec.index[0],
                     tileSpec.index[1]);
             });
         }
 
         if (sheetSpec.frames) {
             sheetSpec.frames.forEach(frameSpec => {
-                sprites.define(frameSpec.name, ...frameSpec.rect);
+                sprites.define(frameSpec.style, ...frameSpec.rect);
             });
         }
 
         if (sheetSpec.animations) {
             sheetSpec.animations.forEach(animSpec => {
                 const animation = createAnim(animSpec.frames, animSpec.frameLen);
-                sprites.defineAnim(animSpec.name, animation);
+                sprites.defineAnim(animSpec.style, animation);
             });
         }
 
```

Comment:
Defining tiles should still be using name.

Reply:
Now using `animSpec.name`